### PR TITLE
#6 breakdown pt0 - set configs

### PR DESCRIPTION
**Pull request by ribeirojose**
original PR: https://github.com/cowdao-grants/cow-py/pull/6

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.PHONY: codegen web3_codegen orderbook_codegen subgraph_codegen test lint format remove_unused_imports
+
+codegen: web3_codegen orderbook_codegen subgraph_codegen
+
+# web3_codegen:
+# 	poetry run web3_codegen
```
Comment (mfw78) [inline Makefile:6]: Is this covered in subsequent PRs, or should this comment be removed?

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Welcome to the CoW Protocol Python SDK (cow_py), a developer-friendly Python lib
 
 ## 🐄 Features
 
-- Querying CoW Protocol subgraphs.
+- Querying CoW Protocol subgraph.
```
Comment (mfw78) [inline README.md:11]:
```suggestion
- Querying the CoW Protocol subgraph.
```

File: README.md [hunk 3]
```diff
@@ -44,19 +44,47 @@ print(orders)
 - `contracts/`(TODO): A pasture of Smart contract ABIs for interaction.
 - `order_book/`(TODO): Functions to wrangle orders on the CoW Protocol.
 - `order_signing/`(TODO): Tools for signing and validating orders. Anything inside this module should use higher level modules, and the process of actually signing (ie. calling the web3 function to generate the signature, should be handled in contracts, not here).
-- `subgraphs/`(WIP): GraphQL clients for querying CoW Protocol data.
+- `subgraph/`(WIP): GraphQL client for querying CoW Protocol's Subgraph.
```
Comment (mfw78) [inline README.md:47]:
```suggestion
- `subgraph/`(WIP): GraphQL client for querying CoW Protocol's subgraph.
```

File: README.md [hunk 4]
```diff
@@ -44,19 +44,47 @@ print(orders)
...
+              }
+            }
+            """, variables=dict(days=2)
+            )
+
+data = client.get_data(response)
+pprint(data)
 ```
 
 ### Signing an Order (TODO)
```
Comment (mfw78) [inline README.md:90]: I know that this is a staggered PR, in future, please no todo markers throughout the repo (unless also accompanied by an issue to make sure it's tracked)

**Review by mfw78 — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (ribeirojose) [reply Makefile:6]: not relevant anymore 👍

File: README.md (hunk 4 above)
Comment (ribeirojose) [reply README.md:90]: 👍 removed for now

**Pull request merged**